main/static/home.css

```diff
@@ -26,51 +26,95 @@
 	margin-right: 5px;
 }
 a.home-button:hover {
-	background-color: #333;
-	color: #eee;
+	color: black !important;
```

Comment:
That will require repeating selectors (such as `a.homebutton:hover, #section-projects a.homebutton:hover, #section-join-us a.homebutton:hover`), which is messy but definitely the better solution compared to `!important`. Thanks for catching that.